### ISSUE TITLE
app never launches from IOS emulator

### ISSUE THREAD

**Issue by tskweres** [Appverse/Nativescript-NG2-Showcase#4]
I followed the install instructions for nativescript, tns doctor says all is good. Cloned this repo, and ran:

tns run ios --emulator

the app appears in the emulator, but never launches. If I click it, I see the splash screen, then it crashes and goes back to home screen.

Any ideas? Thanks!

**Comment by orgbx** [comment]
Hi,
It probably has to do with the latest naughty xcode update. 
I'll try to upgrade to Angular 2 final and nativescript-angular these days and retest everything to check if it's solid.
Cheers!

**Comment by tskweres** [comment]
awesome thanks!

**Comment by tskweres** [comment]
When I run it through Xcode there's this error:

NativeScript loaded bundle file:///System/Library/Frameworks/UIKit.framework
NativeScript loaded bundle file:///System/Library/Frameworks/Foundation.framework
****\* Fatal JavaScript exception - application has been terminated. *****
Native stack trace:
1   0x100427d48 -[TNSRuntime executeModule:]
2   0x100009434 main
3   0x181c1a8b8 <redacted>
JavaScript stack trace:
1   @file:///app/tns_modules/zone.js/dist/zone-node.js:958:21
JavaScript error:
file:///app/tns_modules/zone.js/dist/zone-node.js:958:21: JS ERROR Error: Could not find module 'timers'. Computed path '/var/containers/Bundle/Application/96DC278D-9CCB-4B1D-8195-DC3F7417B391/NGShowcase.app/app/tns_modules/timers'.

**Comment by tskweres** [comment]
#https://github.com/NativeScript/nativescript-angular/issues/444

This solved it:

In package.json:
#
-    "zone.js": "^0.6.12"
-    "zone.js": "0.6.12"

**Comment by aecz** [comment]
Nice find. I would pinned it to zone.js@0.6.17 as per NativeScript suggestion.

**Comment by orgbx** [comment]
Hi, 
Those issues with zone.js have been resolved in the next versions. 
An upgrade was necessary so I've just updated the whole showcase to latest versions (angular 2.0.0 final and nativescript-angular 1.0.0) -> see #5 
I did not test all components yet but it seems to be running correctly on all platforms.
If you happen to encounter any issue, I will be more than happy to check it again ;)

Will do more tests whenever I can.
Cheers!
